tests/worker/topics/test_delayed_topic.py

```diff
@@ -0,0 +1,100 @@
+import typing as t
+
+from collections.abc import AsyncIterator
+from collections.abc import Awaitable
+from datetime import datetime
+from datetime import timedelta
+from unittest import mock
+from unittest.mock import AsyncMock
+from unittest.mock import call
+
+import asyncstdlib as alib
+import pytest
+
+from saturn_engine.core import TopicMessage
+from saturn_engine.utils import utcnow
+from saturn_engine.worker.services.manager import ServicesManager
+from saturn_engine.worker.services.rabbitmq import RabbitMQService
+from saturn_engine.worker.topics import DelayedTopic
+from tests.utils.rabbitmq import ensure_clean_queue
+
+TEST_DELAY: t.Final[int] = 5
+
+
+@pytest.fixture
+async def topic_maker(
```

Comment:
could make this fixture reusable between the tests by renaming it to `rabbitmq_topic_maker` and add a `topic_cls` parameter to `maker`? You would just need to change for:

```python
        options = topic_cls.Options(
            queue_name=queue_name, **kwargs
        )
        topic = topic_cls(options, services=services_manager.services)
```

`delay` can be passed through `kwargs`